src/ESPEasy.ino

```diff
@@ -1012,3 +1021,45 @@ void backgroundtasks()
   runningBackgroundTasks=false;
   STOP_TIMER(BACKGROUND_TASKS);
 }
+
+#ifdef HAS_ETHERNET
+void ETHEvent(WiFiEvent_t event)
```

Comment:
Similar function for WiFi also has its own .ino file
So please move this to some file named `Eth_events.ino` or maybe a more appropriate file name.

Reply:
I will move this to
ESPEasy**ETH**Event.ino as it is called ESPEasy**Wifi**Event.ini in Wifi case.

Reply:
Change pushed to https://github.com/kretzp/ESPEasy/tree/feature/ethernet_support